ra/ra.go

```diff
@@ -520,8 +520,29 @@ func (ra *RegistrationAuthorityImpl) NewAuthorization(ctx context.Context, reque
 			}
 		}
 	}
+	if features.Enabled(features.ReusePendingAuthz) {
+		nowishNano := ra.clk.Now().Add(time.Hour).UnixNano()
+		identifierTypeString := string(identifier.Type)
+		pendingAuth, err := ra.SA.GetPendingAuthorization(ctx, &sapb.GetPendingAuthorizationRequest{
+			RegistrationID:  &regID,
+			IdentifierType:  &identifierTypeString,
+			IdentifierValue: &identifier.Value,
+			ValidUntil:      &nowishNano,
+		})
+		if berrors.Is(err, berrors.NotFound) {
```

Comment:
I don't think we use this style of having an empty if body anywhere else. It seems like it would be clearer written as:
```
if err != nil && !berrors.Is(err, berrors.NotFound) {
  // internal server error
} else if err == nil { 
  // return pending auth
}
// Carry on with creation
```

Thoughts?